### PR TITLE
Feature/triplestore

### LINKED ISSUE #360: [S6 -> S7 port] Implement triplestore (but decide which one first)
We could port over the Marmotta stuff from the current SA@OSU or we could port over the work done by @revgum and use Blazegraph as the triple store.

### PR DESCRIPTION
**Pull request by revgum**
fixes #360

### CODE REVIEW

File: config/blazegraph/blazegraph.production.properties [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#
```
Comment (straleyb) [inline config/blazegraph/blazegraph.production.properties]: What are these .properties files for? Can we not check them into the repo and get them on the production server through scp? Or are these necessary to have in the repo?

File: lib/tasks/blazegraph.rake [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+BLAZEGRAPH_URL = ENV['SCHOLARSARCHIVE_BLAZEGRAPH_URL'] || 'http://localhost:9999/blazegraph'
 BLAZEGRAPH_SPARQL = "#{BLAZEGRAPH_URL}/namespace/#{rails_env}/sparql"
-BLAZEGRAPH_CONFIG_LOG4J = ENV['CVM_BLAZEGRAPH_CONFIG_LOG4J'] || "./config/blazegraph/log4j.properties"
-BLAZEGRAPH_CONFIG_DEFAULT = ENV['CVM_BLAZEGRAPH_CONFIG_DEFAULT'] || "./config/blazegraph/blazegraph.properties"
-BLAZEGRAPH_CONFIG_PRODUCTION = ENV['CVM_BLAZEGRAPH_CONFIG_PRODUCTION'] || "./config/blazegraph/blazegraph.production.properties"
+BLAZEGRAPH_CONFIG_LOG4J = ENV['SCHOLARSARCHIVE_BLAZEGRAPH_CONFIG_LOG4J'] || "./config/blazegraph/log4j.properties"
+BLAZEGRAPH_CONFIG_DEFAULT = ENV['SCHOLARSARCHIVE_BLAZEGRAPH_CONFIG_DEFAULT'] || "./config/blazegraph/blazegraph.properties"
+BLAZEGRAPH_CONFIG_PRODUCTION = ENV['SCHOLARSARCHIVE_BLAZEGRAPH_CONFIG_PRODUCTION'] || "./config/blazegraph/blazegraph.production.properties"
 
 namespace :scholars_archive do
```
Comment (straleyb) [inline lib/tasks/blazegraph.rake]: Would this make the rake task 
```
rake scholars_archive:blazegraph:reset
```

File: config/blazegraph/blazegraph.production.properties (hunk 1 above)
Comment (revgum) [inline config/blazegraph/blazegraph.production.properties]: These are necessary for BG to work correctly for the indexing file structure and logging.. we need them locally for development, but these are reference/basic examples. Implementation on the server will be similar, but should be configured seperately, ran with 'god', and more tuned properties files.

File: lib/tasks/blazegraph.rake (hunk 2 above)
Comment (revgum) [inline lib/tasks/blazegraph.rake]: Yes. Check out lib/tasks/scholars_archive.rake on this branch to see the basic "setup" task that puts Fedora, Solr, and Blazegraph all into motion.

**Pull request merged**